net.rocketpool/image/Dockerfile

```diff
@@ -14,4 +14,4 @@ COPY . .
 RUN npm run build
 
 ENTRYPOINT ["/usr/bin/tini", "--"]
-CMD ["node", "dist/app.js"]
+CMD ["node", "--es-module-specifier-resolution=node", "."]
```

Comment:
when looking up this parameter - the first result seems to indicate that it's [deprecated](https://stackoverflow.com/questions/74660824/nodejs-v19-drops-support-for-es-module-specifier-resolution-node-which-makes-i)?

Reply:
Since we have a locked down version of nodejs (18.X.X) I thought this would be enough.

On the other hand, I can migrate (it's easy) to the latest stable version of nodejs.

**Q:** What do you think about this?

Reply:
yeah, if you can bump the base image to latest node that would be great!

Reply:
Done